oddbird/Metecho
# Track Task-related activities

![](https://source.unsplash.com/featured/?cute,animal&2112141)






## Steps to test/reproduce
- Perform actions on a Task (assignee changes, scratch org creation, reviews, etc)
- Visit `/admin/api/taskactivity/`
- Notice a TaskActivity instance is created for each action being performed

**Note: you will need to configure GitHub webhooks as explained in CONTRIBUTING.rst for the "Submit pull request" and "Merge pull request" events to be tracked propertly.** 

## Show me
![image](https://user-images.githubusercontent.com/2391102/146036766-205573f0-0013-4bfa-8b9c-89fc1e1d1f5b.png)
